diff --git a/usbdrv.h b/usbdrv_mouse.h
index e766173..77d8f4a 100644
--- a/usbdrv.h
+++ b/usbdrv_mouse.h
@@ -749,7 +749,7 @@ typedef struct usbRequest{
 #define USBDESCR_HID_REPORT     0x22
 #define USBDESCR_HID_PHYS       0x23
 
-//#define USBATTR_BUSPOWER        0x80  // USB 1.1 does not define this value any more
+#define USBATTR_BUSPOWER        0x80  // USB 1.1 does not define this value any more
 #define USBATTR_SELFPOWER       0x40
 #define USBATTR_REMOTEWAKE      0x20
 
